app/logic/events.py

```diff
@@ -501,3 +506,18 @@ def getEventRsvpCount(eventId):
         Returns the number of RSVP'd participants for a given eventId.
     """
     return len(EventRsvp.select().where(EventRsvp.event_id == eventId))
+
+def copyRsvp(priorEvent, newEvent):
```

Comment:
Please leave comments on this code. Confused :'(

Reply:
I left a docstring.